app/tests/spec/lib/fxa-client.js

```diff
@@ -695,22 +699,100 @@ define(function (require, exports, module) {
         var token = 'token';
         var code = 'code';
 
+        var relier = {
+          isSync: function () {
+            return true;
+          },
+          wantsKeys: function () {
+            return true;
+          }
+        };
+
         sinon.stub(realClient, 'passwordForgotVerifyCode', function () {
           return p({
             accountResetToken: 'reset_token'
           });
         });
 
         sinon.stub(realClient, 'accountReset', function () {
-          return p(true);
+          return p({
+            email: trim(email),
```

Comment:
`accountReset` doesn't return the email associated with reset. The response looks something like this

```json
{
  "uid": "4c352927cd4f4a4aa03d7d1893d950b8",
  "sessionToken": "27cd4f4a4aa03d7d186a2ec81cbf19d5c8a604713362df9ee15c4f4a4aa03d7d",
  "keyFetchToken": "7d1893d950b8cd69856a2ec81cbfd7d1893d950b3362df9e56a2ec81cbf19d5c",
  "authAt": 1392144866,
  "verified": true
}
```